packages/amplify-graphql-conversation-transformer/src/resolvers/list-messages-init-resolver-fn.template.js

```diff
@@ -0,0 +1,8 @@
+export function request(ctx) {
+  ctx.stash.metadata.index = 'gsi-ConversationMessage.conversationId.createdAt';
```

Comment:
Do we care that the GSI name doesn't include the base type?

Reply:
The generated type `ConversationMessage<RouteName>`? Nope, or at least I don't. It's fine that the GSIs across multiple routes (DDB tables) have the same name.